docs/introduction/quickstart.rst

```diff
@@ -35,7 +35,6 @@ Once you are logged in, you can list all accessible quantum processing units and
   "device_id": 9,
   "device_name": "Galadriel Qblox rack",
   "status": "online",
-  "availability": "available",
```

Comment:
It's not related to your change, but the format looks wrong:
![image](https://github.com/qilimanjaro-tech/qiboconnection/assets/353447/a148985f-2c1e-4eaf-8327-82ce0d3f764c)

Could you format this block as a code-block?